cmd/up/exec.go

```diff
@@ -214,6 +214,10 @@ func (eg *envsGetter) getEnvs(ctx context.Context) ([]string, error) {
 
 	envs = append(envs, eg.getDefaultLocalEnvs()...)
 
+	for _, env := range eg.dev.Environment {
```

Comment:
can we have a tests covering this?

Reply:
928f11577a8c9184d686db338d7998a06473943f